app/views/coronavirus_form/offer_transport.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :title do %><%= t('coronavirus_form.offer_transport.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.offer_transport.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "coronavirus_form/validation_error" %>
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-offer-transport",
```

Comment:
Question, do we even need this line if we shouldn't have analyitics?

Reply:
I've kept it in as we have it for the others, since we may need analytics.